docker-compose.yml

```diff
@@ -0,0 +1,476 @@
+      - frontend-network
+    # --import-realm and --hostname-port are required
+    command: >
+      start-dev
+      --import-realm
+      --hostname-port=8180
+
+  # ==================================================================================
+  # File Management (Song, Score, Kafka, Zookeeper, Minio)
+  # ==================================================================================
+  
+  # ==========================
+  # Zookeeper
+  # ==========================
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+      - "2181:2181"
+    healthcheck:
+      test: ["CMD-SHELL", "echo srvr | nc 127.0.0.1 2181 | grep Mode"]
+      interval: 10s
+      timeout: 10s
+      retries: 3
+      start_period: 20s
+    networks:
+      - backend-network
+
+  # ==========================
+  # Kafka
+  # ==========================
+  kafka:
+    image: confluentinc/cp-kafka:5.4.0
+    container_name: kafka
+    depends_on:
+      zookeeper:
+        condition: service_healthy
+    ports:
+      - "29092:29092"
+      - "9092:9092"
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092,PLAINTEXT_HOST://kafka:9092
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: kafka:9092
+      CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:2181
+    healthcheck:
+      test: ["CMD", "kafka-broker-api-versions", "--bootstrap-server", "localhost:9092"]
+      interval: 10s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+    networks:
+      - backend-network
+
+  # ==========================
+  # Song Database
+  # ==========================
+  song-db:
+    image: postgres:11.1
+    container_name: song-db
+    environment:
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: admin123
+      POSTGRES_DB: songDb
+    volumes:
+      - ./persistentStorage/data-song-db:/var/lib/postgresql/data
+    networks:
+      - backend-network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U admin -d songDb"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
+
+  # ==========================
+  # Song
+  # ==========================
+  song:
+    image: ghcr.io/overture-stack/song-server:438c2c42
+    container_name: song
+    platform: linux/amd64
+    depends_on:
+      keycloak:
+        condition: service_started
+      song-db:
+        condition: service_healthy
+      kafka:
+        condition: service_healthy
+      zookeeper:
+        condition: service_healthy
+    ports:
+      - "8080:8080"
+    environment:
+      # Spring Variables
+      SPRING_PROFILES_ACTIVE: prod,secure,score-client-cred,kafka
```

Comment:
Remove `score-client-cre` profile to allow use score access Token Authentication